crypto/evp/pmeth_gn.c

```diff
@@ -172,8 +172,9 @@ int EVP_PKEY_gen(EVP_PKEY_CTX *ctx, EVP_PKEY **ppkey)
 
         if (keydata == NULL)
             goto not_supported;
-        ret = evp_keymgmt_gen_set_template(ctx->keymgmt,
-                                           ctx->op.keymgmt.genctx, keydata);
+        if (ctx->keymgmt->gen_set_template != NULL)
+            ret = evp_keymgmt_gen_set_template(ctx->keymgmt,
+                                               ctx->op.keymgmt.genctx, keydata);
```

Comment:
This is superceded by #11550

Reply:
This has been kept as a separate commit, so easy to drop.  The CI's break without it.

Unfortunately, I just approved #11550 despite the typo, so I'll be playing catchup.